airbyte-workers/src/main/java/io/airbyte/workers/WorkerApp.java

```diff
@@ -232,9 +251,8 @@ private void registerSync(final WorkerFactory factory) {
 
     final PersistStateActivityImpl persistStateActivity = new PersistStateActivityImpl(statePersistence, featureFlags);
 
-    final Worker syncWorker = factory.newWorker(TemporalJobType.SYNC.name(), getWorkerOptions(maxWorkers.getMaxSyncWorkers()));
-    syncWorker.registerWorkflowImplementationTypes(SyncWorkflowImpl.class);
-    syncWorker.registerActivitiesImplementations(replicationActivity, normalizationActivity, dbtTransformationActivity, persistStateActivity);
+    final Worker worker = factory.newWorker(dataPlane, getWorkerOptions(maxWorkers.getMaxSyncWorkers()));
+    worker.registerActivitiesImplementations(replicationActivity, normalizationActivity, dbtTransformationActivity, persistStateActivity);
```

Comment:
The only thing I am unsure of here is the `persistStateActivity`. That activity takes the state that was output by the replication activity and writes it to the Airbyte config db. In the real implementation of this, will the data plane workers have write access to our Airbyte dbs?

Reply:
Ah yeah this is a known issue and we're planning on rewriting activities that require database access to calling APIs instead. So the activity will still be executed in a data plane but it will be an API call to a server in the control plane rather than a direct database write

Reply:
I see. That seems like a bit of an expansion of scope of the data plane from the original plan, as I believe we originally wanted the data plane to only be pulling jobs off of the intermediate queue. Maybe not a huge deal but wanted to call it out

Reply:
This was included in the original plan, there's some detail on it in Davin's original [Multi-Region Cloud/Data Plane architecture doc](https://docs.google.com/document/d/1hF9uYW4SlQ5GrJ8As9BsuxETQ5uFw7fSlg4j5Sry0d8/edit#heading=h.o4h82qp7kaj1). 

Every sync activity method today relies on direct database access, not just persistState, so we're going to need to replace multiple database calls with API calls